src/modules/organization/organization.service.ts

```diff
@@ -87,10 +87,20 @@ export class OrganizationService {
    * returns single Org with matching namespace
```

Comment:
Is this change to method description correct?
```suggestion
   * Returns Orgs with given owner.
   * Also returns the suborgs of the org and, optionally, associated apps and/or roles
```